eval/src/vespa/eval/eval/simple_value_engine.h

```diff
@@ -0,0 +1,37 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "tensor_engine.h"
+
+namespace vespalib {
+namespace eval {
+
+/**
+ * This is a TensorEngine implementation for the SimpleValue
+ * reference implementation.
+ **/
+class SimpleValueEngine : public TensorEngine
```

Comment:
I do not think we actually need this when we also have the immediate generic implementations wired through the EngineOrFactory class.

Where we would use SimpleValueEngine, we can just use EngineOrFactory(SimpleValueBuilderFactory::get()) instead.